src/TimingData.cpp

```diff
@@ -83,6 +83,265 @@ TimingData::~TimingData()
 	Clear();
 }
 
+enum
+{
+	FOUND_WARP,
+	FOUND_WARP_DESTINATION,
+	FOUND_BPM_CHANGE,
+	FOUND_STOP,
+	FOUND_DELAY,
+	FOUND_STOP_DELAY, // we have these two on the same row.
+	FOUND_MARKER,
+	NOT_FOUND
+};
```

Comment:
Personal reminder: see if enum flags can be used here at some point. If not that, at least an enum class. We have a lot of old code that could use this treatment...